tools/ci/check_grep.sh

```diff
@@ -205,6 +205,12 @@ fi;
 
 if [ "$pcre2_support" -eq 1 ]; then
 	section "regexes requiring PCRE2"
+	part "empty variable values"
+	if $grep -PU '{\n\t},' $map_files; then
```

Comment:
i thought we were using some python shitter for this now

Reply:
unfortunately maplint won't work for this because maplint parses the TGM format (to respect map formatting) and expects proper TGM formatted variables- we have to use check_grep.sh here because this only happens in a completely improper TGM format. we still also do check for other map-related stuff in check_grep.sh (like the five "forbidden" variable edits such as `matrix` and `newlist`) since i presume those can't accurately be finagled within the scope of maplint.

(there are also lints for commented out lines in a TGM file in check_grep.sh)

wish we could use the python stuff, but it's just not built for it